mysqld_exporter.go

```diff
@@ -258,11 +258,11 @@ func (e *Exporter) scrape(ch chan<- prometheus.Metric) {
 	}
 	defer db.Close()
 
-	// limit connections to recommended value
-	db.SetMaxOpenConns(3)
-	db.SetMaxIdleConns(3)
-	// close idle connections after max lifetime
-	db.SetConnMaxLifetime(5 * time.Minute)
+	// by design exporter should use maximum one connection at any given time
```

Comment:
Please capitalize the sentence and end with a `.`.   Also maybe "one connection per request" might be more correct.